### ISSUE TITLE
should not change cwd when there are multiple splits

### ISSUE THREAD

**Issue by winkee01** [ahmedkhalf/project.nvim#100]
Vim supports `:lcd` for local windows and `:cd` for global windows. but when I switch between splits (which have different project roots), this plugin just keeps changing **cwd**, this is a redundant operation. So basically, when you execute `:Telescope projects` to open a file from a project in a new split, you should only call `:lcd` and when you switch between different splits, there is no need to call it again, vim supports **local cwd** for each window.

**Comment by vintingb** [comment]
I agree with u.  When I start a go project which rely on a  module=off package, if i type `gd` to know some method or struct in the package , gopls alway error with not a go module package

**Comment by DrKJeff16** [comment]
What is your `scope_chdir` set to?
